docs/index.html

```diff
@@ -489,15 +501,18 @@ <h5>SSH Agent integration</h5>
 
         <dt id="faq-ssh-agent-pageant"><a href="#faq-ssh-agent-pageant">Why is Pageant refusing my keys?</a></dt>
         <dd>
-            Pageant does not support confirm-on-use or automatic removal of key after a timeout. There doesn't seem to be
-            any alternative to Pageant for Windows that supports both of them.
+            Pageant does not support confirm-on-use or automatic removal of key after a timeout.
```

Comment:
Has this changed? AFAIK OpenSSH for Windows does not support confirm-on-use or automatic removal after a timeout.

Reply:
That's a fair question, I'll verify that tomorrow!